core/templates/services/playthrough.service.ts

```diff
@@ -274,24 +298,48 @@ export class PlaythroughService {
    * If none of the issue types have been discovered, returns null instead.
    */
   private createNewPlaythrough(): Playthrough | null {
-    if (this.misTracker && this.misTracker.foundAnIssue()) {
-      return this.playthroughObjectFactory
-        .createNewMultipleIncorrectSubmissionsPlaythrough(
-          this.explorationId, this.explorationVersion,
-          this.misTracker.generateIssueCustomizationArgs(),
-          this.recordedLearnerActions);
-    } else if (this.cstTracker && this.cstTracker.foundAnIssue()) {
-      return this.playthroughObjectFactory
-        .createNewCyclicStateTransitionsPlaythrough(
-          this.explorationId, this.explorationVersion,
-          this.cstTracker.generateIssueCustomizationArgs(),
-          this.recordedLearnerActions);
-    } else if (this.eqTracker && this.eqTracker.foundAnIssue()) {
-      return this.playthroughObjectFactory
-        .createNewEarlyQuitPlaythrough(
-          this.explorationId, this.explorationVersion,
-          this.eqTracker.generateIssueCustomizationArgs(),
-          this.recordedLearnerActions);
+    // TODO(#15212): Remove the below check once codebase is strictly typed.
+    if (
+      this.explorationId &&
+      this.explorationVersion &&
+      this.recordedLearnerActions
+    ) {
+      if (
+        this.misTracker &&
+        this.misTracker.foundAnIssue()
```

Comment:
Is there a reason you're changing these to multiple lines when that wasn't the case before?

In general if you can keep the previous code the same (especially if there's no strong reason to change it), that makes it easier to see the diff.

Reply:
Done, not found any strong reason behind that.